<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoginWindow</class>
    <widget class="QDialog" name="LoginWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>380</width>
                <height>220</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Login Sistem</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="labelTitle">
                    <property name="text">
                        <string>Silakan Login</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">font-size:18px; font-weight:bold;</string>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="labelUsername">
                            <property name="text">
                                <string>Username:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="input_username" />
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="labelPassword">
                            <property name="text">
                                <string>Password:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="input_password">
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="buttonLayout">
                    <item>
                        <widget class="QPushButton" name="btnLogin">
                            <property name="text">
                                <string>Login</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btnCancel">
                            <property name="text">
                                <string>Batal</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>